cclib/method/ddec.py

```diff
@@ -121,6 +121,22 @@ def calculate(self, indices=None, fupdate=0.05):
             )
         )
 
+        # Notify user about the total charge in the density grid
+        integrated_density = self.charge_density.integrate()
+        self.logger.info(
+            "Total charge density in the grid is {}. If this does not match what is expected, using a finer grid may help.".format(
+                integrated_density
+            )
+        )
+
+        # Notify user about the total charge in the density grid
+        integrated_density = self.charge_density.integrate()
+        self.logger.info(
+            "Total charge density in the grid is {}. If this does not match what is expected, using a finer grid may help.".format(
+                integrated_density
+            )
+        )
+
```

Comment:
Both of these should be removed, since the check already exists on line 108. Make sure to remove the already-existing extra one too.

Reply:
I've removed these. Thanks for letting me know!